app/views/players/_form.html.erb

```diff
@@ -9,7 +9,7 @@
   </div>
   <div class="form-group">
     <%= f.label :age %>
-    <%= f.text_field :age, class: 'form-control', placeholder: "Enter age" %>
+    <%= f.number_field :age, class: 'form-control', placeholder: "Enter age" %>
```

Comment:
Age will become out-of-date as time passes. Consider storing date of birth or year of birth as this won't need to be updated and age can be derived from that.